QueueAndStack/Program.cs

```diff
@@ -0,0 +1,101 @@
+         * удаляется из очереди, операция GetMaxValue очереди должна отражать новый самый большой сохраненный элемент.*/
+        public static Queue<int> AddValuesOfTheQueue()
+        {
+            while (true)
+            {
+                Console.WriteLine("Enter int values separated by comma and a space: ");
+                var value = Console.ReadLine();
+                try
+                {
+                    var convertedValue = value.Split(", ").ToList();
+                    var nums = new Queue<int>();
+                    foreach (var item in convertedValue)
+                    {
+                        nums.Enqueue(int.Parse(item));
+                    }
+                    Console.WriteLine($"The maximum element of the queue is: {GetMaxValue(nums)}");
+
+                    return nums;
+                }
+                catch (FormatException)
+                {
+                    Console.WriteLine("You've entered invalid value");
+                }
+            }
+        }
+
+        public static int GetMaxValue(Queue<int> queue)
+        {
+            return queue.Max();
+        }
+
+        public static Queue<int> DeleteValueInTheQueue(Queue<int> queueToDeleteFrom)
+        {
+            Console.WriteLine("Enter the value to delete: ");
+            var value = Convert.ToInt32(Console.ReadLine());
+            queueToDeleteFrom = new Queue<int>(queueToDeleteFrom.Where(x => x != value)) { };
+            Console.WriteLine($"The maximum element of the queue is: {GetMaxValue(queueToDeleteFrom)}");
+
+            return queueToDeleteFrom;
+        }
+
+        /* Task 2. Напишите программу, которая принимает на вход три буквы и отображает их в обратном порядке.
+         * Используйте Stack.*/
+        public static Stack<char> AddValuesOfTheStack()
+        {
+            while (true)
+            {
+                Console.WriteLine("Enter 3 char values separated by comma and a space: ");
+                var value = Console.ReadLine();
+                try
+                {
+                    var convertedValue = value.Split(", ").ToList();
+                    var chars = new Stack<char>();
+                    foreach (var item in convertedValue)
+                    {
+                        if (convertedValue.IndexOf(item) > 2)
+                        {
+                            Console.WriteLine("You've entered more than 3 characters!");
+                            break;
+                        }
+                        chars.Push(Convert.ToChar(item));
+                    }
+
+                    return chars;
+                }
+                catch (FormatException)
+                {
+                    Console.WriteLine("You've entered invalid value!");
+                }
+            }
+        }
+
+        public static void SortStackDesc(Stack<char> stack)
+        {
+            var orderedList = stack.OrderByDescending(x => x).ToList();
```

Comment:
The task is to write in console in reverse ordering of the input order. E.g. you've entered: q, a, z. Expected result - z, a, q

Reply:
😞 Fixed.